src/applications/vaos/tests/new-appointment/components/PreferredDatePage.unit.spec.jsx

```diff
@@ -1,133 +1,145 @@
+      'What is the earliest date you’d like to be seen?',
     );
+    expect(screen.baseElement).to.contain.text(
+      'Please pick a date within the next 13 months.',
+    );
+  });
 
-    form.find('form').simulate('submit');
+  it('should not submit empty form', async () => {
+    const store = createTestStore(initialState);
+    const screen = renderWithStoreAndRouter(<PreferredDatePage />, {
+      store,
+    });
 
-    expect(form.find('.usa-input-error').length).to.equal(1);
-    expect(history.push.called).to.be.false;
-    form.unmount();
+    fireEvent.click(screen.getByText(/Continue/));
+    expect(await screen.findByRole('alert')).to.contain.text(
+      'Please provide a response',
+    );
+    expect(screen.history.push.called).to.be.false;
   });
 
-  it('it should not submit with past date', () => {
-    const openFormPage = sinon.spy();
-    const updateFormData = sinon.spy();
-    const history = {
-      push: sinon.spy(),
-    };
-
-    const form = mount(
-      <PreferredDatePage
-        openFormPage={openFormPage}
-        updateFormData={updateFormData}
-        history={history}
-        data={{ preferredDate: '2016-02-02' }}
-      />,
+  it('it should not submit with past date', async () => {
+    const store = createTestStore(initialState);
+    const screen = renderWithStoreAndRouter(<PreferredDatePage />, {
+      store,
+    });
+
+    userEvent.selectOptions(screen.getByRole('combobox', { name: /month/i }), [
+      '2',
+    ]);
+    userEvent.selectOptions(screen.getByRole('combobox', { name: /day/i }), [
+      '2',
+    ]);
+    await userEvent.type(
+      screen.getByRole('spinbutton', { name: /year/i }),
+      '2016',
     );
-
-    form.find('form').simulate('submit');
-
-    expect(form.find('.usa-input-error').length).to.equal(1);
-    expect(history.push.called).to.be.false;
-    form.unmount();
+    fireEvent.click(screen.getByText(/Continue/));
+    expect(await screen.findByRole('alert')).to.contain.text(
+      'Please enter a future date ',
+    );
+    expect(screen.history.push.called).to.be.false;
   });
 
-  it('it should not submit beyond 395 days into the future', () => {
-    const openFormPage = sinon.spy();
-    const updateFormData = sinon.spy();
-    const history = {
-      push: sinon.spy(),
-    };
-
-    const form = mount(
-      <PreferredDatePage
-        openFormPage={openFormPage}
-        updateFormData={updateFormData}
-        history={history}
-        data={{ preferredDate: '2050-02-02' }}
-      />,
+  it('it should not submit beyond 395 days into the future', async () => {
+    const store = createTestStore(initialState);
+    const screen = renderWithStoreAndRouter(<PreferredDatePage />, {
+      store,
+    });
+
+    userEvent.selectOptions(screen.getByRole('combobox', { name: /month/i }), [
+      '2',
+    ]);
+    userEvent.selectOptions(screen.getByRole('combobox', { name: /day/i }), [
+      '2',
+    ]);
+    await userEvent.type(
```

Comment:
The docs suggest that we only need to use `await` if we're adding a delay in typing, so you could probably remove this.

Reply:
@jbalboni I thought that too based on the docs but the IDE throws a warning about needing it 
![image](https://user-images.githubusercontent.com/8315447/98696143-6f2d0580-2341-11eb-92fe-cd94835377f8.png)